jobserver/settings.py

```diff
@@ -259,6 +262,16 @@
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
 
+# CSP
+# https://django-csp.readthedocs.io/en/latest/configuration.html
+CSP_DEFAULT_SRC = ["'self'"]
+CSP_IMG_SRC = ["*", "data:"]
```

Comment:
Why '*'? Is that something that will be changed in future?

Reply:
Also, the avatars coming from github seem to be of type "text/html / Redirect". Not sure if they'll be picked up under the img_src section?